pkger/models.go

```diff
@@ -939,6 +958,16 @@ func (c colors) influxViewColors() []influxdb.ViewColor {
 	return iColors
 }
 
+func (c colors) strings() []string {
+	clrs := []string{}
```

Comment:
is it desired to set a `[]` if no colors provided? or does the `nil` value better suffice? When these get json marshaled the one will provide an empty array, whereas the other will be a `null`.

Reply:
are these api contracts outlined anywhere? i'm curious what the FE folks are expecting, or if either will suffice.

Reply:
think you're cool to leave this as is for now. Should probably check the swagger docs for this 🤔, I'm not sure about teh contract's existence